instance/instance.go

```diff
@@ -0,0 +1,139 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+// Package instance implements VM instance software configs.
+package instance
```

Comment:
Just move this into the policies package, call it local.go or something. No need to make a separate package.
I think the entry point should just be policies.Run, then it gathers api guestpolicies, parses a local config file, runs merge, then passes it all through the exist apply flow.
Also we can keep all the functions private.

Reply:
Done. This also made `golint` (which I just found about :slightly_smiling_face: ) happier.